### PR TITLE
implement ocean albedo calculation

### LINKED ISSUE #91: Implement ocean albedo calculation for UFS
Currently the SWnet calculated for the ocean assumes a constant albedo of 0.06 for both direct and diffuse SW radiation. This albedo does not vary with sun angle. This results in an excess of SW radiation to the ocean. See Issue https://github.com/ufs-community/ufs-weather-model/issues/1768 for more details.

The ocean albedo calculation can be implemented by adding the needed run phase and having UFS call the ocean albedo calculation in ``med_phases_prep_ocn``. Currently this calculation happens only for CESM, but can be generalized for non-cesm use without duplication of code.

### PR DESCRIPTION
**Pull request by DeniseWorthen**
### Description of changes

Implements ocean albedo calculation for UFS.

### Specific notes

CMEPS Issues Fixed #91 

Are changes expected to change answers? (specify if bfb, different at roundoff, more substantial) 

This will change answers in UFS for all cpld and datm tests. 

- related to https://github.com/ufs-community/ufs-weather-model/pull/1773

### CODE REVIEW

File: .github/workflows/srt.yml [hunk 1]
```diff
@@ -26,8 +26,8 @@ jobs:
       CPPFLAGS: "-I/usr/include -I/usr/local/include "
       LDFLAGS: "-L/usr/lib/x86_64-linux-gnu -lnetcdf -lnetcdff -lpnetcdf"     
       # Versions of all dependencies can be updated here
-      ESMF_VERSION: v8.4.0
-      PARALLELIO_VERSION: pio2_5_10
+      ESMF_VERSION: v8.5.0
+      PARALLELIO_VERSION: pio2_6_0
```
Comment (junwang-noaa) [inline .github/workflows/srt.yml:30]: Does this PR require ESMF 850 and pio260?

Comment (DeniseWorthen) [reply .github/workflows/srt.yml:30]: No, those are not required.

**Review by uturuncoglu — APPROVED** [review]
@DeniseWorthen It looks good to me.

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]

**Comment by FernandoAndrade-NOAA** [comment]
Testing on #[1773 ](https://github.com/ufs-community/ufs-weather-model/pull/1773) has completed successfully, please continue with the merge process for this PR.

**Pull request merged**